devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy

```diff
@@ -71,7 +71,7 @@ metadata {
 	}
 
 	preferences {
-		input "tempOffset", "number", title: "Temperature offset", description: "Select how many degrees to adjust the temperature.", range: "*..*", displayDuringSetup: false
```

Comment:
`Select how many degrees to adjust the temperature.` is the correct `description`